src/core/utils/geometryutils.h

```diff
@@ -87,6 +87,9 @@ class QFIELD_CORE_EXPORT GeometryUtils : public QObject
 
     //! Returns an empty (i.e. null) point
     static Q_INVOKABLE QgsPoint emptyPoint() { return QgsPoint(); }
+
+    //! Returns an point
+    static Q_INVOKABLE QgsPoint point( const double &x, const double &y ) { return QgsPoint( x, y ); }
```

Comment:
```suggestion
    static Q_INVOKABLE QgsPoint point( double x, double y ) { return QgsPoint( x, y ); }
```

Reply:
@m-kuhn , why removing the const and reference?

Reply:
https://stackoverflow.com/a/2582813/2319028

> Passing arguments by value and thus copying them can be expensive - const references avoid that expensive step while still promising the caller that the object won't be changed.
> 
> Usually fundamental types (int, double, ...) are passed by value, while class-types are passed by const reference.

Reply:
Thanks for the lesson, master.